linebot/models/messages.py

```diff
@@ -71,6 +74,18 @@ def __init__(self, id=None, text=None, emojis=None, **kwargs):
         else:
             self.emojis = emojis
 
+        if mention:
+            mentionees = []
+            for mentionee in mention["mentionees"]:
```

Comment:
@nanato12 maybe you can not use `mention["mentionees"]` first because the developer would use like this:

```
Mention(mentionee=[Mentionee(....)])
``` 

If we judge here first, maybe `self.mention` just equal `mentionees` would be ok.

Any questions please feel free to let me know :)

Reply:
Already like that.
Is this wrong?

Reply:
oh, it's my fail.